libtvm/tvm_htab.c

```diff
@@ -41,7 +41,13 @@ static inline unsigned int htab_hash(const char *k, const unsigned int size)
 {
 	unsigned int hash = 1;
 
-	char *c; for (c = (char *)k; *c; c++)
```

Comment:
Although the original code is not the clearest, I don't think this is an objective improvement.